pol-core/pol/module/osmod.cpp

```diff
@@ -1400,6 +1460,215 @@ BObjectImp* OSExecutorModule::mf_GetEnvironmentVariable()
   }
 }
 
+BObjectImp* OSExecutorModule::mf_SendEmail()
+{
+  if ( Core::settingsManager.email_cfg.url.empty() )
+    return new BError( "Email settings are not configured" );
+
+  Core::UOExecutor& this_uoexec = uoexec();
+
+  if ( this_uoexec.pChild == nullptr )
```

Comment:
`pChild` is never set here and I think never needed, since the instruction doesn't get rerun just suspended and revived.

Reply:
Addressed in 0574bee2ee8a56c83cb66797038da63cc026255c